api/openapi.gen.yaml

```diff
@@ -242,6 +244,12 @@ components:
           type: string
         instance_type:
           type: string
+        names:
+          items:
+            type: string
+          type: array
+        poweroff:
+          type: boolean
```

Comment:
I realized that the PowerOff field was missing in AWS response, so I added it.

Reply:
Do we want to expose it? given it's an internal flag, that we do not want to expose to customers, I'd rather hide it (on request as well as response) :thinking: